test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/ControllerUnitTestabilityTests.cs

```diff
@@ -34,6 +34,7 @@ public void ControllerView_InvokedInUnitTests(object model, string viewName)
             var viewResult = Assert.IsType<ViewResult>(result);
             Assert.Equal(viewName, viewResult.ViewName);
             Assert.NotNull(viewResult.ViewData);
+            Assert.Same(model, viewResult.Model);
```

Comment:
Please also remove the useless block at lines 42-46. Same for the other test you're changing. This is going to come up every time someone looks at these tests.